src/main/java/io/javalin/plugin/openapi/OpenApiPlugin.kt

```diff
@@ -0,0 +1,42 @@
+package io.javalin.plugin.openapi
+
+import io.javalin.Javalin
+import io.javalin.core.plugin.Plugin
+import io.javalin.core.plugin.PluginLifecycleInit
+import io.javalin.core.util.OptionalDependency
+import io.javalin.core.util.Util
+import io.javalin.plugin.openapi.ui.ReDocRenderer
+import io.javalin.plugin.openapi.ui.SwaggerRenderer
+
+/**
+ * Plugin for the the automatic generation of an open api schema.
+ * The schema can be extracted with [JavalinOpenApi.createSchema].
+ */
+class OpenApiPlugin(
+        private val options: OpenApiOptions
+) : Plugin, PluginLifecycleInit {
```

Comment:
Same as previous comment. Is there a particular reason I'm not seeing to do it like this?